template-parts/pagination.php

```diff
@@ -5,8 +5,8 @@
  * @package shiro
  */
 
-$newer = get_theme_mod( 'wmf_pagination_newer', __( 'Newer', 'shiro-admin' ) );
-$older = get_theme_mod( 'wmf_pagination_older', __( 'Older', 'shiro-admin' ) );
+$newer = get_theme_mod( 'wmf_pagination_newer', __( 'Next', 'shiro-admin' ) );
+$older = get_theme_mod( 'wmf_pagination_older', __( 'Previous', 'shiro-admin' ) );
```

Comment:
Thanks for raising this @goldenapples.

The ticket asks:
> (...) ideally buttons for the main search results and pages filter would show "Previous" "Next" instead of "Older" and "Newer" - which should only display on the post filtered results.

Reading it again, I think that it makes sense to:
- Replace "Newer" with "Previous"
- Replace "Older" with "Next"

Change applied.